core/controllers/story_viewer.py

```diff
@@ -207,6 +209,34 @@ def post(self, story_id, node_id):
                 learner_at_review_point_in_story or learner_completed_story):
             ready_for_review_test = True
 
+        # If there is no next_node_id, the story is marked as completed else
+        # mark the story as incomplete.
+        if not next_node_id:
```

Comment:
Just to confirm, in the completed case, this is None, right? And not some other value.

Reply:
Yes, it's None.